plugins/woocommerce/src/Blocks/BlockTypes/ProductButton.php

```diff
@@ -189,6 +191,21 @@ protected function render( $attributes, $content, $block ) {
 				data-wc-layout-init="callbacks.syncTemporaryNumberOfItemsOnLoad"
 			';
 
+			$wrapper_attributes = get_block_wrapper_attributes(
+				array(
+					'class' => implode(
+						' ',
+						array_filter(
+							[
+								'wp-block-button wc-block-components-product-button',
+								esc_attr( $text_align_styles_and_classes['class'] ?? '' ),
```

Comment:
Is there a reason why we added this logic, even if `text_align_styles_and_classes` is never defined
?

I noticed this by reviewing #50662

Reply:
Good catch 👍 that may of been a copy/paste mistake, I can spin up a separate PR to remove it.

Reply:
No worries @louwie17! I think that we can fix it in  https://github.com/woocommerce/woocommerce/pull/50662

Reply:
@gigitux I spun up a PR: https://github.com/woocommerce/woocommerce/pull/51729